srcpkgs/qjackctl/template

```diff
@@ -3,7 +3,7 @@ pkgname=qjackctl
 version=0.5.5
 revision=1
```

Comment:
please bump revision from 1 to 2

Revision needs to be bumped whenever a change that affects the end-users is made (adding patches, changing build configuration options, deleting files)